src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,7 +4,9 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
+
 public class Bulldozer extends Machine {
```

Comment:
Task: "You have abstract class Machine and three sub-classes: Bulldozer, Excavator and Truck. Feel free to add some type-specific fields to these classes." Lets add some type-specific fields to all Machine classes